deps/media-playback/media-playback/media.c

```diff
@@ -267,6 +291,21 @@ static inline int64_t mp_media_get_next_min_pts(mp_media_t *m)
 			min_next_ns = m->a.frame_pts;
 	}
 
+	if( m->enable_caching ) {
```

Comment:
If cache is enabled, the above shouldn't be executed no?

Reply:
code above works correctly when it is a first run when caching not finished.

Reply:
Correct, it shouldn't cause issues.